client/app/lib/components/MaterialSummernote.jsx

```diff
@@ -131,6 +134,7 @@ class MaterialSummernote extends React.Component {
         />
         <div className="material-summernote">
           <ReactSummernote
+            ref={(ref) => { this.reactSummernote = ref; }}
```

Comment:
It should be a callback function that will be called after the react component is mounted to pass the reference back